vizro-core/tests/unit/vizro/models/test_page.py

```diff
@@ -153,5 +153,4 @@ def test_page_build_left_side_removed(standard_px_chart):
     page = vm.Page(title="Single Page", components=[vm.Graph(id="scatter_chart", figure=standard_px_chart)])
     dashboard = vm.Dashboard(pages=[page])
     dashboard.pre_build()
-    dashboard.navigation.pre_build()
-    assert "className='left_side'" not in str(page.build())
+    assert "className='hidden'" in str(page.build())
```

Comment:
I don't think this is correct? If we don't test for the left_side anymore, we should test for the nav_panel being removed? The hidden className can also be in the page.build independently if the left-side is being removed or not.

```suggestion
    assert "className='hidden'" in str(page.build())
```